graylog2-server/src/test/java/org/graylog2/shared/utilities/ExceptionUtilsTest.java

```diff
@@ -71,4 +71,33 @@ public void hasCauseOf_returnsTrueIfTheCauseIsSubtypeOfTheProvidedType() {
         assertThat(ExceptionUtils.hasCauseOf(
                 new RuntimeException("parent", new IOException("asdasd")), IOException.class)).isTrue();
     }
+
+    @Test
+    public void getShortenedStackTrace() {
```

Comment:
I believe some other test cases might be out there :D

Reply:
care to elaborate?